wasmbin_fuzz/fuzz/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+
+[package]
+name = "wasmbin-fuzz"
+version = "0.0.0"
+authors = ["Automatically generated"]
+publish = false
+edition = "2018"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = "0.4.2"
+wasmbin = "0.5.1"
+
+# Prevent this from interfering with workspaces
+[workspace]
+members = ["."]
+
+[[bin]]
+name = "decode"
+path = "fuzz_targets/decode.rs"
+
+[[bin]]
+name = "roundtrip"
+path = "fuzz_targets/roundtrip.rs"
```

Comment:
Seeing as the file `roundtrip.rs` was deleted, should we also remove this target?

Reply:
yep, you´re right! deleted now